src/makielayout/blocks/axis.jl

```diff
@@ -369,6 +369,19 @@ function initialize_block!(ax::Axis; palette = nothing)
         end
     end
 
+    _xticksmirrored = lift(mirror_ticks, xaxis.tickpositions, ax.xticksize, ax.xtickalign, Ref(scene.px_area), :x, ax.xaxisposition[])
```

Comment:
could we try to make this one linesegment call, by just creating one point array for the linesegments?
Also, whats the meaning of `_` as a prefix in this context?

Reply:
I'd like to avoid using `_` for scenarios where no better name was found... But it's just a nitpick, not a deal breaker ^^

Reply:
I can change it now, I only used keyword args when developing to avoid a struct redefinition.

Reply:
I've changed the underscore names now, the combining of linesegments would be nice but I'm not motivated right now